components/drops/calendar/DropsCalendar.vue

```diff
@@ -15,6 +15,7 @@
         <div class="mb-6 flex items-center">
           <NeoButton
             variant="secondary-rounded"
+            root-class="!flex-shrink-0"
```

Comment:
was thinking if we should add this to the button css file itself

Reply:
no need, I think